lib/google_search_data_viewer/keywords/schemas/keyword_upload.ex

```diff
@@ -0,0 +1,28 @@
+defmodule GoogleSearchDataViewer.Keywords.Schemas.KeywordUpload do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias GoogleSearchDataViewer.Accounts.Schemas.User
+
+  schema "keyword_uploads" do
+    field :name, :string
+    field :html, :string
+
+    field :status, Ecto.Enum,
+      values: [:pending, :inprogress, :completed, :failed],
+      default: :pending
+
+    belongs_to :user, User
+
+    timestamps()
+  end
+
+  def changeset(keyword_upload \\ %__MODULE__{}, attrs) do
+    keyword_upload
+    |> cast(attrs, [:name, :user_id])
+    |> put_change(:status, :pending)
```

Comment:
Can we omit this function as we already define the default value in the schema?

Reply:
If you are referring to `put_change(:status, :pending)` , yes I guess we could. Do you have a preference on setting the default value in the schema or the changeset?

Reply:
Yes, for me the schema is better because we can have multiple changesets